store/mongo/datastore_mongo.go

```diff
@@ -325,6 +328,21 @@ var (
 		Options: mopts.Index().
 			SetName(IndexDeviceDeploymentsLogs),
 	}
+
+	// 1.2.13
+	IndexArtifactProvides = mongo.IndexModel{
+		Keys: bson.D{
+			{Key: model.StorageKeyImageProvidesKey,
+				Value: 1},
+			{Key: model.StorageKeyImageProvidesValue,
+				Value: 1},
```

Comment:
Which queries will match this index?
Please note that you are indexing key and value separately, and not combined.
In order to query for specific key/vaule we'll have to use $elemMatch.

Reply:
the query will be in the enterprise version of the repo;
the idea [there](https://github.com/mendersoftware/deployments/pull/795#issuecomment-1340510348) was to store provides as a list of embedded documents, and yes, in this case the query will use $elemMatch (I should probably use it in the test, so it would more clear how I'm planning to use it)